src/Http/Controllers/AppointmentController.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace LibreEHR\FHIR\Http\Controllers;
+
+use Illuminate\Http\Request;
+use LibreEHR\FHIR\Adapters\FHIRAppointmentAdapter;
+use Illuminate\Support\Facades\DB;
+use phpDocumentor\Reflection\DocBlock\Tags\Return_;
+
+class AppointmentController extends AbstractController
+{
+    public function __construct()
+    {
+        $this->adapter = new FHIRAppointmentAdapter;
```

Comment:
This is not needed. Use the IOC binding in Utilities/Providers/FHIRServiceProvider. Generally it's bad practice to instanciate objects in the constructor, dependencies should always be passed in as parameters.